src/web/src/pages/CourseScheduler.vue

```diff
@@ -457,6 +462,19 @@ export default {
 };
 </script>
 
+<!-- All Styling For Dark Mode For Course Information Modals.  -->
+<style scoped>
+/deep/ .onStyleDarkMode {
+  background: var(--dark-primary);
+  color: white;
+}
+
+/deep/ .offStyleDarkMode {
```

Comment:
no longer need this

Reply:
Sounds good. I thought I removed it. Will be fixed.

Reply:
Update: This Has Been Removed Through Latest Commit.